deploy/eck-beats/lint-values.yaml

```diff
@@ -0,0 +1,6 @@
+# Values required for `helm lint` to succeed
+# Currently no default is set in values.yaml
+# until https://github.com/helm/helm/pull/11440
+# is merged
+spec:
+  daemonSet: {}
```

Comment:
Do we need this or change how we run `helm lint`?

Because running the linter without it doesn't fail there is just an extra INFO level log.

```
> helm lint --strict .; echo $?
engine.go:176: [INFO] Missing required value: A Beat type is required
engine.go:189: [INFO] Fail: At least one of daemonSet or deployment is required for a functional Beat
==> Linting .

1 chart(s) linted, 0 chart(s) failed
0
```

Reply:
@thbkrkr I swore mine was failing without this in place.. I'll verify/

Reply:
Yeah the `Fail` confused me, I think, and I didn't check the return value.  I've removed this.